PersistentCache/Src/Alachisoft.NCache.PersistenceStore/Model/CacheStoreDbContext.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) 2019 Alachisoft
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//    http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Cosmos.Storage.Internal;
+using PersistenceProviders.Model;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PersistentNCache.Model
+{
+    public class CacheStoreDbContext:DbContext
+    {        
+        public DbSet<StoreItem> CacheItem { get; set; }
+        public DbSet<DataTypeItem> dataTypeItem{ get; set; }
+
+        string _connectionString = "";
+        DbType dbType = DbType.SqlServer;
+
+        string _serviceEndPoint;
+        string _authKey;
+        string _databaseName;
+
+        public CacheStoreDbContext()
+        {
+            _connectionString = "Data Source =.; Initial Catalog = NCacheDb; Integrated Security = false; User ID = sa; password=4Islamabad";
```

Comment:
why use default value from code.  if required then get value from user . because db context value is always change

Reply:
default constructor is removed as it was never used unless migrations are needed.